docs/advanced.rst

```diff
@@ -739,278 +740,401 @@ Thus, for a higher performance, it can be appropriate to configure the reliabili
+| **C++**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp              |
+|    :language: c++                                       |
+|    :start-after: //CONF-DISCOVERY-LEASE-ANNOUNCEMENT    |
+|    :end-before: //!--                                   |
++---------------------------------------------------------+
+| **XML**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml               |
+|    :language: xml                                       |
+|    :start-after: <!-->CONF-DISCOVERY-LEASE-ANNOUNCEMENT |
+|    :end-before: <!--><-->                               |
++---------------------------------------------------------+
+
+.. _simple_disc_settings:
+
+SIMPLE Discovery Settings
+=========================
+
+The SIMPLE discovery protocol resolves the establishment of the end-to-end connection between various RTPS entities
+communicating via the RTPS protocol.
+Fast-RTPS implements the SIMPLE discovery protocol to provide compatibility with the
+`RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_.
+The specification splits up the SIMPLE discovery protocol into two independent protocols:
+
+- **Simple Participant Discovery Protocol (SPDP):** specifies how Participants discover each other in the network; it
+  announces and detects the presence of participants in a domain.
+
+- **Simple Endpoint Discovery Protocol (SEDP):** defines the protocol adopted by the discovered participants for the
+  exchange of information in order to discover the RTPS entities contained in each of them, i.e. the writer and
+  reader Endpoints.
+
++--------------------------+-----------------------------------------------------------------------+
+| Name                     | Description                                                           |
++==========================+=======================================================================+
+| `Initial Announcements`_ | It defines the behavior of the RTPSParticipant initial announcements. |
++--------------------------+-----------------------------------------------------------------------+
+| `Simple EDP Attributes`_ | It defines the use of the SIMPLE protocol as a discovery protocol.    |
++--------------------------+-----------------------------------------------------------------------+
+
+
+.. _`Initial Announcements`:
+
+Initial Announcements
+---------------------
+
+`RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_ simple discovery mechanism requires the participant to
+send announcements. These announcements are not deliver in a reliable fashion, and can be disposed of by the network. In
+order to avoid the discovery delay induced by message disposal, the initial announcement can be set up to make several
+shots, in order to increase proper reception chances.
```

Comment:
chances -> changes?

Reply:
It's actually chance in the sense of opportunity of receiving a message.